scripts/gee

```diff
@@ -5763,6 +5767,11 @@ function _print_color_block() {
   printf " %s%3d%s" "$(tput setaf "${FG}"; tput setab "${C}")" "$C" "${_COLOR_RST}"
 }
 
+function gee__set_main_by_asking_github() {
+  _set_main_by_asking_github
+  _info "MAIN=$MAIN"
+}
+
```

Comment:
Is this function used anywhere?

Reply:
ah, a backdoor function for testing github.  I'll remove it.  Removed.

Reply:
(all functions named with the "gee__cmdname" are commands that can be invoked by running "gee cmdname"